cves/CVE-2009-1195.yml

```diff
@@ -32,7 +32,11 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  The Options=IncludesNOEXEC in configuration for Apache HTTP Server is not handled
+  properly on the AllowOverride directive. This allows a local user to execute code
```

Comment:
It might be helpful to explain how the option is not handled properly. And what these options are intended to do.